float64 distance_meters   # Request message: Distance in (m) to be converted to feet
---
float64 distance_feet     # Response message: Distance in feet after conversion
bool success              # Response message type
